lib/Chat/ChatManager.php

```diff
@@ -531,4 +531,37 @@ public function deleteMessages(Room $chat): void {
 	public function searchForObjects(string $search,  array $objectIds, string $verb = '', int $offset = 0, int $limit = 50): array {
 		return $this->commentsManager->searchForObjects($search, 'chat', $objectIds, $verb, $offset, $limit);
 	}
+
+	public function addGroupNotify(array $results, string $search, Room $room, Participant $participant): array {
```

Comment:
Let's call this:
```suggestion
	public function addConversationNotify(array $results, string $search, Room $room, Participant $participant): array {
```

as group notifications could refer to real Nextcloud groups in the future (now that groups can be added to conversations).